app/assets/javascripts/components/overview/details.jsx

```diff
@@ -222,10 +222,10 @@ const Details = React.createClass({
         </div>
         <div className="module__data extra-line-height">
           {instructors}
-          {online}
-          {campus}
-          {staff}
-          {school}
+          {Features.wikiEd ? online : null}
+          {Features.wikiEd ? campus : null}
+          {Features.wikiEd ? staff : null}
+          {Features.wikiEd ? school : null}
```

Comment:
We should keep `school` (which will renders as Institution for P & E programs.) Otherwise, users will not be able to change that if needed.